bin/verify_aws_cli_setup

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -eu
+
+echo "--- :aws: Ensuring AWS CLI is installed and configured..."
+
+echo "~~~ Checking if AWS CLI is installed..."
+if ! command -v aws &> /dev/null; then
+  echo "[!] AWS CLI not found."
+  exit 1
+else
+  echo "AWS CLI found at $(which aws)."
+  aws --version
+fi
+
+echo "~~~ Checking if AWS credentials are configured in env vars or config files..."
+
+if [ -n "${AWS_ACCESS_KEY_ID:-}" ]; then
+    echo "AWS_ACCESS_KEY_ID found in environment variables."
+else
+    echo "No AWS_ACCESS_KEY_ID in environment variables."
+fi
+
+if [ -n "${AWS_SECRET_ACCESS_KEY:-}" ]; then
+    echo "AWS_SECRET_ACCESS_KEY found in environment variables."
+else
+    echo "No AWS_SECRET_ACCESS_KEY in environment variables."
+fi
+
+AWS_CONFIG_PATH="$HOME/.aws/config"
+if [ -f "$AWS_CONFIG_PATH" ]; then
+    echo "AWS config file exists at $AWS_CONFIG_PATH."
+    echo "Printing config via aws configure list:"
+    aws configure list
+else
+    echo "No AWS config file found at $AWS_CONFIG_PATH."
+fi
+
+AWS_CREDENTIALS_PATH="$HOME/.aws/credentials"
+if [ -f "$AWS_CREDENTIALS_PATH" ]; then
+    echo "AWS credentials file exists at $AWS_CREDENTIALS_PATH."
+    echo "Printing profiles via aws configure list:"
+    aws configure list-profiles
+else
+    echo "No AWS credentials file found at $AWS_CREDENTIALS_PATH."
+fi
```

Comment:
Worth noting that it is totally valid for an EC2 instances to not have `~/.aws/config`, `~/.aws/credentials`, nor env vars to provide AWS access, because those EC2 instances living in AWS infra are provided their credentials via roles and policies instead. I guess that's why you didn't `exit 1` in the `else` of those cases 👍  but just wanted to highlight that fact for other reviewers too.